posthog/batch_exports/http.py

```diff
@@ -95,35 +95,21 @@ class BatchExportRunViewSet(TeamAndOrgViewSetMixin, viewsets.ReadOnlyModelViewSe
     queryset = BatchExportRun.objects.all()
     serializer_class = BatchExportRunSerializer
     pagination_class = RunsCursorPagination
+    filter_rewrite_rules = {"team_id": "batch_export__team_id"}
 
-    def get_queryset(self, date_range: tuple[dt.datetime, dt.datetime] | None = None):
-        if not isinstance(self.request.user, User) or self.request.user.current_team is None:
-            raise NotAuthenticated()
-
-        if date_range:
-            return self.queryset.filter(
-                batch_export_id=self.kwargs["parent_lookup_batch_export_id"],
-                created_at__range=date_range,
-            ).order_by("-created_at")
-        else:
-            return self.queryset.filter(batch_export_id=self.kwargs["parent_lookup_batch_export_id"]).order_by(
-                "-created_at"
-            )
-
-    def list(self, request: request.Request, *args, **kwargs) -> response.Response:
-        """Get all BatchExportRuns for a BatchExport."""
-        if not isinstance(request.user, User) or request.user.team is None:
-            raise NotAuthenticated()
+    def get_queryset(self):
+        queryset = super().get_queryset()
```

Comment:
We always want to call the super value unless we are really going off piste (for example doing CH queries or something